lib/manager/gradle-wrapper/artifacts.ts

```diff
@@ -3,25 +3,35 @@ import Git from 'simple-git/promise';
 import { resolve } from 'path';
 import * as fs from 'fs-extra';
 import { logger } from '../../logger';
-import { UpdateArtifact, UpdateArtifactsResult } from '../common';
+import {
+  UpdateArtifact,
+  UpdateArtifactsConfig,
+  UpdateArtifactsResult,
+} from '../common';
 import { exec, ExecOptions } from '../../util/exec';
-import { readLocalFile } from '../../util/fs';
 import { platform } from '../../platform';
 import { VERSION_REGEX } from './search';
-import { gradleWrapperFileName, prepareGradleCommand } from '../gradle/index';
+import { gradleWrapperFileName, prepareGradleCommand } from '../gradle';
+import { readLocalFile } from '../../util/fs';
 
 async function addIfUpdated(
+  config: UpdateArtifactsConfig,
```

Comment:
```suggestion
```

Reply:
fixed in https://github.com/renovatebot/renovate/pull/5876/commits/e307e2da378e9d02cfcbdba2b01c7593f79a10e2